components/sup/src/main.rs

```diff
@@ -365,32 +334,10 @@ fn sub_start(m: &ArgMatches) -> Result<()> {
     let running = Manager::is_running(&cfg)?;
 
     try!(command::start::run(cfg.clone(), maybe_spec.clone(), maybe_local_artifact));
-    if !running {
-        return Ok(());
-    }
-    if let Some(spec) = maybe_spec {
-        match wait_for_status(cfg, spec.ident.clone(), &ProcessState::Up) {
-            Ok(status) => {
-                match status.process.state {
-                    ProcessState::Up => {
-                        outputln!("The {} service was succesfully started with PID {}",
-                                  spec.ident,
-                                  status.process.pid.unwrap());
-                    }
-                    _ => {
-                        outputln!("The {} service was not succesfully started. Current state: {}",
-                                  spec.ident,
-                                  status.process.state);
-                        std::process::exit(1);
-                    }
-                }
-            }
-            Err(e) => {
-                outputln!("The {} service was not succesfully started: {}",
-                          spec.ident,
-                          e);
-                std::process::exit(1);
-            }
+    if running {
+        if let Some(spec) = maybe_spec {
+            outputln!("The supervisor is starting the {} service. See supervisor output for more details.",
```

Comment:
heh. that was my first rev and I removed it. Its the better call and I'll put it back.